bindings/java/CMakeLists.txt

```diff
@@ -256,29 +254,81 @@ if(NOT OPEN_FOR_IDE)
   add_dependencies(fat-jar copy_lib)
   add_dependencies(packages fat-jar)
 
-  if(RUN_JAVA_TESTS)
-    set(enabled ENABLED)
-  else()
-    set(enabled DISABLED)
-  endif()
   set(TEST_CP ${tests_jar} ${target_jar})
-  message(STATUS "TEST_CP ${TEST_CP}")
-  add_java_test(NAME DirectoryTest CLASS_PATH ${TEST_CP}
-    CLASS com.apple.foundationdb.test.DirectoryTest ${enabled})
 
-  if(RUN_JUNIT_TESTS)
+  if(RUN_JUNIT_TESTS OR RUN_JAVA_INTEGRATION_TESTS)
+    # We use Junit libraries for both JUnit and integration testing structures, so download in either case
```

Comment:
We should try to move everything to junit5. There are many advantages there.

Reply:
Easy enough. done.